lib/asciidoctor/latex/converter.rb

```diff
@@ -89,35 +89,47 @@
 module Asciidoctor::LaTeX
   module Html5ConverterExtensions
 
-    def environment node
+    ENV_CSS = "+++<div style='line-height:1.5em;font-size:1.05em;font-style:oblique;margin-bottom:1.5em'>+++"
+    DIV_END = '+++</div>+++'
+    TABLE_ROW_END = '+++</tr></table>+++'
+
+    def info node
+
+      attrs =  node.attributes
+
+      warn "\n    node: #{node.node_name}".cyan
+      warn "   attrs: #{attrs}".cyan
+      warn "   title: #{attrs['title']}".cyan
+      warn "    role: #{attrs['role']}".cyan
+      warn "   level: #{attrs['level']}".cyan
+      warn " options: #{attrs['options']}".cyan
+      warn " type   : #{attrs['type']}".cyan
+      warn " caption: #{node.caption}".red
+      warn "      id: #{attrs['id']}".cyan
+      warn " content: #{node.content}".blue
```

Comment:
I’ve been pushing my changes to my fork of asciidoctor-latex as @mojavelinux
advised me to do.  How do I pull your changes into my fork. (Once they are
there I can pull them back to my laptop).  Or is this the wrong way to proceed?


Jim

http://noteshare.io



> On Jan 5, 2015, at 5:08 PM, Jakub Jirutka <notifications@github.com> wrote:
> 
> In lib/asciidoctor/latex/converter.rb <https://github.com/asciidoctor/asciidoctor-latex/pull/18#discussion-diff-22493675>:
> 
> > +    TABLE_ROW_END = '+++</tr></table>+++'
> > +
> > +    def info node
> > +
> > +      attrs =  node.attributes
> > +
> > +      warn "\n    node: #{node.node_name}".cyan
> > +      warn "   attrs: #{attrs}".cyan
> > +      warn "   title: #{attrs['title']}".cyan
> > +      warn "    role: #{attrs['role']}".cyan
> > +      warn "   level: #{attrs['level']}".cyan
> > +      warn " options: #{attrs['options']}".cyan
> > +      warn " type   : #{attrs['type']}".cyan
> > +      warn " caption: #{node.caption}".red
> > +      warn "      id: #{attrs['id']}".cyan
> > +      warn " content: #{node.content}".blue
> I’ve been actually trying to refactor logging, but stopped in middle because from my point of view, most of these log messages seems to be unnecessary even for the debug level.
> 
> Therefore, could you please consider which messages are actually useful, mainly for users of the backend, and remove the rest?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/asciidoctor/asciidoctor-latex/pull/18/files#r22493675>.
>